mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.purgeOrder = async (req, res) => {
+  try {
+    const result = await Order.findOneAndRemove({ _id: req.params.orderId });
+    if (!result) {
+      return res.status(404).json({ message: "Order not located" });
+    }
+    res.status(200).json({ message: "Order eradicated successfully" });
+  } catch (err) {
```

Comment:
The `catch` block should be more specific about the error it's catching. Instead of just catching the generic `err`, it should catch a specific error type, such as `Error` or a custom error type. This will make the error handling more robust and informative.